templates/patterns/banners/banner_hero.ui_patterns.yml

```diff
@@ -14,13 +19,21 @@ banner_hero:
       label: "Title"
       description: "Hero title"
       preview: "White Paper on the future of Europe"
-    url:
```

Comment:
We shouldn't remove old params, we should create the link in the template. and not here.